package.json

```diff
@@ -113,6 +116,12 @@
     "cover:unit": "node --max-old-space-size=4096 --harmony --trace-deprecation ./node_modules/istanbul/lib/cli.js cover --report none node_modules/mocha/bin/_mocha -- test/*.unittest.js",
     "cover:report": "istanbul report",
     "cover:report-min": "istanbul report --report lcovonly",
-    "publish-patch": "npm run lint && npm run beautify-lint && mocha && npm version patch && git push && git push --tags && npm publish"
+    "publish-patch": "npm run lint && npm run beautify-lint-all && mocha && npm version patch && git push && git push --tags && npm publish"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint",
+      "beautify-lint"
```

Comment:
I think this name should match the npm task name to be run ( `beautify-lint-all` )

Reply:
This should be `beautify-rewrite`, no?

Reply:
@pavelloz no. It is the bin. I don’t to lint the entire codebase but the staged files (specified using arguments)

@probablyup no because I just want to lint the files and fail. The original PR was modifying the files but there were concerns. The aim here is to force the contributor to lint her work before committing and prevent CI failures